lib/appLauncher.js

```diff
@@ -5,11 +5,8 @@ const { existsSync } = require('fs');
 const { get } = require('http');
```

Comment:
I think downloadManager.js is also included in @tizentv/tools.
But in webIDE-common-tizentv, downloadManager.js is included as well, and even used for downloading certification resources.
Why can't it be united?

Reply:
done